contribution_guide.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to uTensor
+
+Welcome to uTensor contribution guide. Want to make things better? [Talk to us](utensor@googlegroups.com)!
+
+## Ways to Contribute
+
+### Two kinds of pull request
+
+  1. Adding new features to uTensor:
+    -  Create a new [issue](https://github.com/uTensor/uTensor/issues) and discuss the idea with core maintainers. We want to make sure your time is well spent and learn about you.
+    -  Start the implementing the feature once the feedback is received.
+      - uTensor runtime [local build instructions](#localBuild)
+      - Code Generator Developer Build
```

Comment:
We should link to code-gen's ReadMe.   Wait for @dboyliao to merge cgen dev to master?

Reply:
I think I can merge it this night.
Though OOB haven't done yet but I think it's a good time to make exception here.
Or it looks like a dead lock for our parallel workflow :)

Reply:
Nah   that's fine.   We can just link it to the current README. It should get updated as you update the Master.
@Knight-X What do you think?

Reply:
We can just link to current README now and update it in the future. By the way, what is the OOB ?

Reply:
@neil-tan taught be this word before, 'out-of-box' testing.
To me, it looks like a integration test : p

Reply:
hah
It's a bit like the "unboxing" videos on youtube. In an integration test, functionalities are probably the most concern. In an OOB, we care about the experience of the user too.

Reply:
@Knight-X Can you please link the "Code Generator Developer Build" to the current cgen ReadMe?  Pretty sure we can merge this once that's sorted.